lib/onetime/models/metadata.rb

```diff
@@ -77,13 +77,13 @@ def received!
       # Make sure we don't go from :shared to :viewed
       return unless state?(:new) || state?(:viewed)
       @state = :received
-      update_fields :state => :received, :received => Time.now.utc.to_i, :secret_key => nil
+      update_fields :state => :received, :received => Time.now.utc.to_i, :secret_key => "null"
```

Comment:
Let's go with empty strings. Like you mention in your comment, "null" then becomes a *special* string. It's unlikely that we'd ever need to store a literal value of "null" but it's still a potential pitfall.

```suggestion
      update_fields :state => :received, :received => Time.now.utc.to_i, :secret_key => ""
```